### ISSUE TITLE
UI glitch in explore people - text overflows

### ISSUE THREAD

**Issue by gannebamm** [GeoNode/geonode#4376]
If the postion string is to long it will overflow the container in 12er and 8er bootsstrap:
![grafik](https://user-images.githubusercontent.com/6172324/56887883-ff16a180-6a72-11e9-8772-22e3868b522c.png)

On 6er it is working fine (even though it is not 'nice'):
![grafik](https://user-images.githubusercontent.com/6172324/56887906-0dfd5400-6a73-11e9-8105-11b42095d2d7.png)

**Comment by t-book** [comment]
This should solve it:

```
.profile-details small {
    word-break: break-word;
}
```

**Comment by gannebamm** [comment]
> 
> 
> This should solve it:
> 
> ```
> .profile-details small {
>     word-break: break-word;
> }
> ```
Almost. Should be word-**wrap** : break-word
```
.profile-details small {
    word-wrap: break-word;
}
```

**Comment by t-book** [comment]
Of course! Thanks for pointing!

**Comment by gannebamm** [comment]
closed by #4380
